HW04/aleksei_zhuchkov/lib/util.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+def print_notification_for_students_and_mentor(name, surname, notifications, type_person)
```

Comment:
Please, store the notifications' logic in the Notification class

Reply:
Hi, I have done it https://github.com/labs-ruby/homeworks-2022/pull/96/commits/2317202f0cf45739eda68d5119b5ad79bb38f418
please check.